<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncWindow</class>
 <widget class="QMainWindow" name="SyncWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SyncWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="8" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>File Progress:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="6">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Password: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="4">
         <widget class="QLineEdit" name="lineUser">
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="3">
         <widget class="QLineEdit" name="lineSyncDir"/>
        </item>
        <item row="8" column="1" colspan="5">
         <widget class="QProgressBar" name="progressFile">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="4">
         <widget class="QLineEdit" name="lineHost"/>
        </item>
        <item row="0" column="0" rowspan="5">
         <widget class="QLabel" name="labelImage">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="owncloud_sync.qrc">:/images/owncloud.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="5">
         <widget class="QProgressBar" name="progressTotal">
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="time"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Update every: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="4">
         <widget class="QLineEdit" name="linePassword">
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="buttonSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Total Progress:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Username: </string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QToolButton" name="buttonSyncDir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Sync Directory: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Hostname: </string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string> seconds</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="6">
         <widget class="QLabel" name="status">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="6">
         <widget class="QPushButton" name="conflict">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Conflicts exist, click to resolve.</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Select the one files that should be used, either the server version or local version:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="columnCount">
           <number>4</number>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Server Modified</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Local Modified</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Which to keep?</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineHost</tabstop>
  <tabstop>lineUser</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>lineSyncDir</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonSyncDir</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="owncloud_sync.qrc"/>
 </resources>
 <connections/>
</ui>
